Algebraic Topology/algebraic-topology.tex

```diff
@@ -248,16 +249,16 @@ \section{Homotopy}
   The above is well defined because
   \begin{itemize}
   \item If $[p]=[q]$, then $[f(p)] = [f(q)]$ because $h.f$ gives a
-    homotopy.  ((What the heck is h?))
+    homotopy.  ((What the heck is h?)) %
   \item $(f_*p)*(f_*q) = f_*(p*q)$
   \end{itemize}
 \end{prop}
 
 \begin{prop}
   Given a fundamental group $\Pi_1(X, x)$, then we can restrict $f_*$
-  to the functor $$f_* : \Pi_1(X, x) to \Pi_1(Y, f(x)).$$
+  to the functor $$f_* : \Pi_1(X, x) \to \Pi_1(Y, f(x)).$$
 
-  picture $x$ (loop) to $f(x)$ (loop)
+  picture $x$ (loop) to $f(x)$ (loop) %
```

Comment:
Maybe we should add the picture? I can screencap pictures from Hatcher if necessary. Just let me know.